test/elastic_record/index_test.rb

```diff
@@ -8,7 +8,10 @@ def test_copy
   end
 
   def test_alias_name
-    assert_equal 'widgets', index.alias_name
```

Comment:
Keep this assertion to show that setting an index_suffix causes the name to change.